<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWindow</class>
 <widget class="QWidget" name="AddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>command:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>icon path:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameText">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>241</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="commandText">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>241</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="iconText">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>241</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>97</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>select icon</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>97</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancleButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>97</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cancle</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
